daemon/daemon.go

```diff
@@ -1088,7 +1088,16 @@ func createPrefixLengthCounter() *counter.PrefixLengthCounter {
 
 // NewDaemon creates and returns a new Daemon with the parameters set in c.
 func NewDaemon() (*Daemon, *endpointRestoreState, error) {
-	// Validate the daemon specific global options
+	// Prepopulate option.Config with options from CLI.
```

Comment:
Please use a function to do all of these. I might even suggest a new file

Reply:
Done.